app/assets/stylesheets/metrics.scss

```diff
@@ -186,7 +186,7 @@
   }
 
   .tenant-filter-bar {
-    margin-left: 5px;
+    margin-left: 0;
     .dropdown-menu.open {
       margin-top: -11px;
     }
```

Comment:
It's a one line change, I do not want to style fix the whole file for this PR :-(

We can have a new PR for fixing style, if we decide it's important enough.